src/pages/ReimbursementAccount/BankInfo/substeps/Plaid.tsx

```diff
@@ -77,7 +77,7 @@ function Plaid({reimbursementAccount, reimbursementAccountDraft, onNext, plaidDa
 
     return (
         // @ts-expect-error TODO: Remove this once Form (https://github.com/Expensify/App/issues/31972) is migrated to TypeScript.
```

Comment:
```suggestion
        // @ts-expect-error TODO: Remove this once FormProvider (https://github.com/Expensify/App/issues/31972) is migrated to TypeScript.
```

Reply:
I'll just change it everywhere in our feature then